src/sparql/TripleBuilder.ts

```diff
@@ -19,6 +20,17 @@ export default class TripleBuilder {
 		};
 	}
 
+	private checkValueType( value: string | UnitValue ): string | number {
+		if ( typeof value === 'string' ) {
+			return value;
+		}
+		try {
+			return value.value;
+		} catch ( e ) {
+			throw new Error( 'Unit Values not yet supported!!' );
+		}
```

Comment:
```suggestion
		throw new Error( 'Unit Values not yet supported!!' );
```

Is there a case where value is not a string but an object that has a value key, and it is not a UnitValue?

Reply:
I tried this first and it was making the test 'with quantity value' in tests/unit/sparql/QueryObjectBuilder.spec.ts always throw the error. But now it works. :)